common/src/test/java/io/seata/common/loader/EnhancedServiceLoaderTest.java

```diff
@@ -34,8 +34,8 @@
      */
     @Test
     public void testLoadByClassAndClassLoader() {
-        Hello load = EnhancedServiceLoader.load(Hello.class, Hello.class.getClassLoader());
-        Assertions.assertEquals(load.say(), "Bonjour");
+        Hello load = EnhancedServiceLoader.load(Hello.class,Hello.class.getClassLoader());
```

Comment:
Should keep format unchanged as before, missing space character here.

Reply:
restored